kani-compiler/src/main.rs

```diff
@@ -16,6 +16,8 @@
 #![feature(f128)]
 #![feature(f16)]
 #![feature(non_exhaustive_omitted_patterns_lint)]
+#![feature(cfg_version)]
+#![cfg_attr(not(version("1.86")), feature(float_next_up_down))]
```

Comment:
```suggestion
// Once the `stable` branch is at 1.86 or later, remove this line, since float_next_up_down is stabilized
#![cfg_attr(not(version("1.86")), feature(float_next_up_down))]
```
Probably don't need this reminder but I figure it doesn't hurt.